androminion/res/values-de/strings-expansions.xml

```diff
@@ -11,7 +11,8 @@
 	<string name="Hinterlands">Hinterland</string>
 	<string name="DarkAges">Dark Ages</string>
 	<string name="Guilds">Die Gilden</string>
-	<string name="Adventures">Abenteuer</string>
+	<string name="Adventures">Adventures</string>
+	<string name="Empires">Empires</string>
```

Comment:
Shouldn't this be something like "Imperien" or "Reiche"? Or are you waiting for the official translation to be released?